tiny_dnn/core/framework/tensor.h

```diff
@@ -268,9 +281,10 @@ class Tensor {
    * with offset zero
    *
    */
-  // TODO
   Tensor subView(std::initializer_list<size_t> const &new_shape) {
-    return Tensor(xt::broadcast(storage_, new_shape));
+    Tensor res = Tensor(xt::broadcast(storage_, storage_.shape()));
```

Comment:
does then the new tensor share storage with the previous or it's just a copy of the view?